src/libcore/num/mod.rs

```diff
@@ -2250,7 +2250,8 @@ assert_eq!(
 );
 ```"),
             #[stable(feature = "int_to_from_bytes", since = "1.32.0")]
-            #[rustc_const_unstable(feature = "const_int_conversion", issue = "53718")]
+            #[rustc_const_stable(feature = "const_int_conversion", since = "1.43.0")]
+            #[allow_internal_unstable(const_transmute)]
             #[inline]
             pub const fn to_ne_bytes(self) -> [u8; mem::size_of::<Self>()] {
                 // SAFETY: integers are plain old datatypes so we can always transmute them to
```

Comment:
Please add a const-safety argument as in https://github.com/rust-lang/rust/pull/63770/files#diff-1decd9f63c23ff5e439882aa3624d87aR66 in the places the `transmute` is used. (Applies generally when `const fn` and `unsafe` is being combined. `const fn`s have additional safety requirements beyond what `fn`s must adhere to, in particular as laid out in https://www.ralfj.de/blog/2018/07/19/const.html (see "CTFE correctness", which is the property which must be preserved).) In this case the safety argument should be simple.

Reply:
@Centril I added const safety comments which are basically a copy of the safety comments inside the functions, I think that's what was required, right?

Reply:
Yep, thank you.